### ISSUE TITLE
Arrow Keys on heatmap Not Functional When Screenreader Active

### ISSUE THREAD

**Issue by longmatthewh** [highcharts/highcharts#20981]
#### Expected behaviour
With a screenreader like Voiceover is active, and focused on a heatmap chart, I should be able to use the arrow keys to navigate through the series points and have the point data/label read aloud for each point.

#### Actual behaviour
With a screenreader like Voiceover is active, and focused on a heatmap chart, I am unable to use the arrow keys to navigate through the series. In some cases it appears that the arrow keys work with smaller data sets and a screenreader active. However, in the JSFiddle below, I've defaulted to a smaller data set, and I still cannot use the arrow keys in the chart with a screenreader active.

This is not an issue when a screenreader is not in use.

#### Live demo with steps to reproduce
https://jsfiddle.net/matt_long/svq1dpnw/4/

#### Product version
Latest Highcharts - Highcharts JS v11.4.1

#### Affected browser(s)
Chrome

**Comment by oysteinmoseng** [comment]
Hi Matt!

Screen readers typically override keyboard shortcuts when they run. Individual screen readers have various mechanisms for passing through keys to the webpage. In other words, this is expected behavior, and screen reader users can interact with the charts through mechanisms familiar to them. This is validated through user testing with screen reader users.

With that said, we are working on a new accessibility model that makes use of custom keyboard commands, and enables multi-dimensional navigation as you are requesting. The initial tests are promising, but in order not to be confusing to the users, a lot of care has to be taken to give instructions as the users navigate. You can expect more functionality in Highcharts later this year and in 2025 in this area.

I'm closing this particular issue for now, but feel free to comment if you have any further questions or feedback.

**Comment by longmatthewh** [comment]
Thank you for the response, but it's unclear to me how a screen reader user is supposed to navigate the heatmap.

**Comment by oysteinmoseng** [comment]
@longmatthewh Sure, let me try to elaborate.

1. Screen reader users will by default, upon reaching the chart, get an overview description of the chart, including the title, chart type, axis information and so on. Once they get to the interactive chart, they should be able to navigate through individual series and data points as they would through any web document, similarly to a list. The points are marked up as `role="img"` with `aria-label`s. With VoiceOver on desktop, you would navigate using `VO+arrow L/R` at this point, and the screen reader should step through each data point linearly in order. By default, series are marked up as region landmarks, allowing the user to navigate between data series more quickly.

2. There is not much I can share in terms of details yet, as we need to do extensive user testing to determine the best interaction models for different types of charts. The goal, however, is to provide smarter defaults that allow more complex charts to be navigated using custom keyboard commands. Of course as with anything Highcharts related, we will make sure the experience is customizable in case you want to make changes for your particular use case.
